swatch-contracts/TEST_PLAN.md

```diff
@@ -1429,3 +1429,115 @@ This section validates capacity calculations across time boundaries. Tests verif
+- **Description**: Verify that a non-metered capacity decreases when the sockets quantity is decreased
+- **Test Steps**:
+  1. Given: create a subscription with Satellite Server product (satellite-server) sockets 8
+  2. When: Update the sockets to be 4 and run Subscription sync
+  3. Then: Verify that the sockets are updated (decreased) and the capacity reflects the updated sku
+- **Expected Results**: 
+  - the sockets will reflect the 4 cores
+
+**capacity-metrics-TC010 - Increasing Non metered sockets**
+- **Description**: Verify that a RHEL for x86 (rhel-for-x86) capacity increases when the sockets quantity is increased
+- **Test Steps**:
+  1. Given: create a subscription with sku sockets 4
+  2. When: Update the Satellite Server product (satellite-server) sockets to be 8 and run Subscription sync/Reconcile (?)
+  3. Then: Verify that the sockets are updated (increased) and the capacity reflects the updated sku
+- **Expected Results**: 
+  - the sockets will reflect the 8 sockets
+
+## Accounts with Subscriptions and or contracts
+**capacity-metrics-TC011 - Increasing subscription on an account with multiple contracts**
+- **Description**: Verify that an Account with multiple contracts and a subscription increases when the capacity quantity is increased
+- **Test Steps**:
+  1. Given: Create an account with two contracts and a subscription
+  2. When: Increase the sku related to the subscription and run Subscription sync
+  3. Then: Verify that the capacity increases
+- **Expected Results**: 
+  - that the capacity increases
+
+**capacity-metrics-TC012 - Decreasing subscription on an account with multiple contracts**
+- **Description**: Verify that an Account with multiple contracts and a subscription decreases when the capacity quantity is decreased
+- **Test Steps**:
+  1. Given: Create an account with two contracts and a subscription
+  2. When: Decrease the sku related to the subscription and run Subscription sync
+  3. Then: Verify that the capacity decreases
+- **Expected Results**: 
+  - that the capacity decreases 
+
+**capacity-metrics-TC013 - Increasing contract on an account with multiple subscriptions**
+- **Description**: Verify that an Account with multiple subscriptions and a contract increases when the capacity quantity is increased
+- **Test Steps**:
+  1. Given: Create an account with two subscriptions and a contract
+  2. When: Increase the metrics related to the contract and run Subscription sync
+  3. Then: Verify that the capacity increases
+- **Expected Results**: 
+  - that the capacity increases
+
+**capacity-metrics-TC014 - Decreasing contract on an account with multiple subscriptions**
+- **Description**: Verify that an Account with multiple subscriptions and a contract decreases when the capacity quantity is decreased
+- **Test Steps**:
+  1. Given: Create an account with two subscriptions and a contract
+  2. When: Decrease the metrics related to the contract and run Subscription sync
```

Comment:
the Subscription sync won't update the contract capacity. This API is only for subscriptions, so the `run Subscription sync` is not necessary here.